02.calendar/cal.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+import dayjs from "dayjs";
+import ja from "dayjs/locale/ja.js";
+
+dayjs.locale(ja);
+const argv = minimist(process.argv.slice(2));
+
+let month = dayjs().format("M");
+let year = dayjs().format("YYYY");
+
+if (argv["m"]) {
+  month = String(argv["m"]);
+  if (argv["y"]) {
+    year = String(argv["y"]);
+  }
+}
```

Comment:
```suggestion
const today = dayjs();
const month = argv["m"] ? String(argv["m"]) : today.format("M");
const year = argv["m"] && argv["y"] ? String(argv["y"]) : today.format("YYYY");
```

なるべくconstを使った方がいいのでこうするとか？
（yearの条件にも`argv["m"]`が出てきてしまうのが若干ネックですが）

Reply:
> なるべくconstを使った方がいいのでこうするとか？

そもそも論でこういう考え方がありませんでした。
教えていただきありがとうございます🙇‍♂️